pyanaconda/modules/payloads/payloads_interface.py

```diff
@@ -30,9 +30,14 @@
 class PayloadsInterface(KickstartModuleInterface):
     """DBus interface for Payload module."""
 
-    def GetActivePayloadPath(self) -> ObjPath:
-        """Get path to the payload which is used now."""
-        return self.implementation.get_active_payload_path()
+    def GetActivePayload(self) -> ObjPath:
+        """Get active payload.
+
+        :raise: PayloadNotSetError if payload is not set
+        """
+        return PayloadContainer.to_object_path(
+            self.implementation.get_active_payload()
+        )
 
     def IsPayloadSet(self) -> Bool:
```

Comment:
Btw. I don't think this method is necessary.  You can always ask for the active payload directly. It can be unsafe to use it, for example to check if we can call `GetActivePayload`, because the value can change between the calls. I don't remember if there was a discussion about it, but there is no `FIXME` comment in the docstring.

Reply:
The same as above, not sure about how it should be handled.

I'll add `FIXME`.